packages/expo-modules-core/android/src/main/java/expo/modules/kotlin/views/ExpoView.kt

```diff
@@ -46,4 +48,12 @@ abstract class ExpoView(
       post(Runnable { measureAndLayout() })
     }
   }
+
+  override fun dispatchDraw(canvas: Canvas) {
+    // When the border radius is set, we need to clip the content to the padding box.
+    // This is because the border radius is applied to the background drawable, not the view itself.
+    // It is the same behavior as in React Native.
+    BackgroundStyleApplicator.clipToPaddingBox(this, canvas)
```

Comment:
@lukmccall  @Kudo  this seems like it forces all views into the behavior of `overflow: hidden`. `View` in RN and `<div>` on web default to `overflow: visible`, compared to some elements like image which do default to hidden.

Is forcing overflow to be hidden in Expo Views intentional?

Reply:
I think it's okay for now. I don't see how that negatively could affect our current use case.

Reply:
Expo Views allow children (can be containers), and 3p can extend, right? 

I can give a few examples where forcing `overflow: hidden` in a container may lead to undesirable effects, from what I have touched recently.
1. A child of the container has a `boxShadow`, or `outline`, which may draw outside the bounds of the parent view. The container hiding overflow would hide this
2. A child has some animation, like a scale or rotation transform, where it goes outside the box
3. A child may use an absolutely positioned child for animated effects. Think, tapping on an image, and a hear floats to the top of the screen.

Reply:
I've now added the ability to disable the `clipToPaddingBox`. Thank you for providing more context.